table.csv

```diff
@@ -35,6 +35,7 @@ multihashes,,
 sha1,               ,                         0x11
 sha2-256,           ,                         0x12
 sha2-512,           ,                         0x13
+dbl-sha2-256,       ,                         0x56
```

Comment:
does this mean double sha2-256? might make sense to also add double sha2-512, cjdns uses that for ipv6 addresses (but on the other hand it truncates to 16 bytes).